pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -106,7 +106,7 @@ def circuit(theta, phi):
         assert np.allclose(qnode(theta, phi), qnode_def(theta, phi), atol=tol(dev.analytic))
         assert np.allclose(grad(theta, phi), grad_def(theta, phi), atol=tol(dev.analytic))
 
-    @pytest.mark.parametrize("ret", [qml.expval, qml.var])
+    @pytest.mark.parametrize("ret", ["expval", "var"])
```

Comment:
Why this change? Did it not work before or is this just better/safer?

Reply:
Tape mode was activated after `ret` got its value, so non-tape `qml.expval` and `qml.var` were pulled in.